data/language/ko-KR.txt

```diff
@@ -48,7 +48,7 @@ STR_0043    :스페이스 링
 STR_0044    :역방향 자유낙하 코스터
 STR_0045    :엘리베이터
 STR_0046    :수직 낙하 롤러코스터
-STR_0047    :현금지급기
+STR_0047    :현금 지급기
```

Comment:
Revert all these language changes. It's probably best to rebase on develop, pick only your own commits and squash them.